blocks/rich-text/format-toolbar/index.js

```diff
@@ -26,19 +27,25 @@ const FORMATTING_CONTROLS = [
 	{
 		icon: 'editor-bold',
 		title: __( 'Bold' ),
+		shortcut: primaryShortcut( 'B' ),
 		format: 'bold',
 	},
 	{
 		icon: 'editor-italic',
 		title: __( 'Italic' ),
+		shortcut: primaryShortcut( 'I' ),
 		format: 'italic',
 	},
 	{
 		icon: 'editor-strikethrough',
 		title: __( 'Strikethrough' ),
+		shortcut: primaryAltShortcut( 'D' ),
```

Comment:
This should be `accessShortcut`, right? https://github.com/WordPress/gutenberg/pull/4411/files#diff-766381e792acf9d4c2505f3aeddcca7eR181

Reply:
Arg, yes, thanks, I think all these code names have confused my brain. Sorry! 😓

Reply:
Yeah, that's why I think two is enough for now. :) https://github.com/WordPress/gutenberg/pull/4411#discussion_r185658847